src/components/ui/Select/select.scss

```diff
@@ -1,47 +1,77 @@
 @import "src/styles/scaffold";
 
-[data-select] {
+[data-fs-select] {
+  // --------------------------------------------------------
+  // Design Tokens for Select
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-select-height                      : var(--fs-spacing-6);
+  --fs-select-min-height                  : var(--fs-control-tap-size);
+  --fs-select-padding                     : var(--fs-spacing-1) var(--fs-spacing-5) var(--fs-spacing-1) var(--fs-spacing-2);
+
+  --fs-select-text-color                  : var(--fs-color-link);
+  --fs-select-border-radius               : var(--fs-border-radius-default);
+
+  --fs-select-bkg                         : transparent;
+  --fs-select-bkg-color-focus             : var(--fs-color-primary-bkg-light);
+  --fs-select-bkg-color-hover             : var(--fs-select-bkg-color-focus);
+
+  --fs-select-transition-timing           : var(--fs-transition-timing);
+  --fs-select-transition-property         : var(--fs-transition-property);
+  --fs-select-transition-function         : var(--fs-transition-function);
+
+  --fs-select-label-color                 : var(--fs-color-text-light);
+  --fs-select-label-margin-right          : var(--fs-spacing-1);
+
+  --fs-select-icon-right                  : var(--fs-spacing-2);
```

Comment:
Not sure, do you think we could be more specific here? `--fs-select-icon-position-right` 🤔

Reply:
right now we don't have this standard in the project, checked to see if somewhere in the code uses this nomenclature, but didn't find any. WDYT @renatamottam ?

Reply:
I like the `--fs-select-icon-position-right` option